src/qml/qgismobileapp.qml

```diff
@@ -2157,6 +2157,7 @@ ApplicationWindow {
       height: 40;
       width: parent.width
       y: parent.height - 112
+      z: 10001
```

Comment:
I understand why. To have it on top. 
But why exactly this value? Just curious.

Reply:
```$ grep -hr " z:" src/qml | sed 's/^\W*//g'
z: 1
z: 10000 // 1000s are embedded feature forms, use a higher value to insure feature form popups always show above embedded feature formes
z: 1
z: 1
z: 1
z: 10
z: 11
z: 12
z: 12
z:20
z: 1
z: 2
z: 10000 // 1000s are embedded feature forms, use a higher value to insure feature form popups always show above embedded feature forms
z: 10000 // 1000s are embedded feature forms, use a higher value to insure feature form popups always show above embedded feature formes
z: 10000 // 1000s are embedded feature forms, use a higher value to insure feature form popups always show above embedded feature forms
z: 1
z: 1000 + embeddedLevel
z: 10001
z: 1```
The 10000 value is used in some of the editorwidgets: ExternalResource, DateTime, RelationEditor